packages/esm-ward-app/src/config-schema.ts

```diff
@@ -37,11 +30,12 @@ export const addressFields = [
 type AddressField = keyof typeof addressFields;
 
 export const configSchema: ConfigSchema = {
-  wardPatientCards: {
-    _description: 'Configure the display of ward patient cards',
-    obsElementDefinitions: {
+  patientCardElements: {
+    _description:
+      'Configuration of various patient card elements. Each configured element must have an unique id, defined in the ward React component being used.',
```

Comment:
Fixing some small grammar:
 `Configuration of various patient card elements. Each configured element must have a unique id, defined in the ward React component being used.`

Reply:
If we could have a way to automatically generate the keys, it would significantly reduce the risk of human error.

Reply:
> If we could have a way to automatically generate the keys, it would significantly reduce the risk of human error.

This might be difficult, with how the ward view can be an extension outside the ward app.